charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -36,15 +36,19 @@ data:
         }
       }
 
+      {{- with .Values.nodeAttestor.k8s.psat }}
```

Comment:
Eventually we can loop all nodeAttestorKeys here. As mentioned below. That's why we should have only one level of nesting in values.

Makes this part simpler for us to manage, and still gives the users a nice consistent API.

Reply:
kk